tools/pipelines/build-client.yml

```diff
@@ -25,7 +25,7 @@ parameters:
 - name: buildToolsVersionToInstall
   displayName: Fluid build tools version (default = installs version in repo)
   type: string
-  default: latest
+  default: "^0.6.0-110101"
```

Comment:
If the scope is azure only, curious why this change here?

Reply:
There is a bug with the latest version of build tools that was affecting this repo. @tylerbutler suggested pinning the version to avoid the issue